src/partials/30_section-features.html

```diff
@@ -5,35 +5,43 @@
         <div class="feature-box active">
           <div class="icon"><i class="fas fa-truck"></i></div>
           <div class="content">
-            <h5>Free shipping</h5>
-            <p>All orders</p>
+            <a href="#">
+              <h5>Free shipping</h5>
+              <p>All orders</p>
+            </a>
           </div>
         </div>
       </div>
       <div class="col">
-        <div class="feature-box">
+        <div class="feature-box active">
```

Comment:
Hej, 
nie podoba mi się to, że dodałeś nową klasę. Spróbuj osiągnąć ten sam efekt ale modyfikując tylko  plik scss

Reply:
Rozwiązanie:
- zgodnie z zaleceniami zmiany w pliku html zostały przywrócone do pierwotnego stanu.
- zostało zmienione w plik _feature-box.scss z '&.active:hover na &.active, &:hover '